src/com/ecfeed/core/generators/algorithms/NwiseScoreEvaluator_RepforTest.java

```diff
@@ -0,0 +1,23 @@
+package com.ecfeed.core.generators.algorithms;
+
+import java.util.List;
+
+import com.ecfeed.core.generators.api.GeneratorException;
+import com.ecfeed.core.generators.api.IConstraintEvaluator;
+import com.ecfeed.core.utils.EvaluationResult;
+
+public class NwiseScoreEvaluator_RepforTest<E> extends NwiseScoreEvaluator_Rep<E> {
```

Comment:
Can be simply called NwiseScoreEvaluatorMock

Reply:
OK. Thanks for the suggestions of naming 👍